### PR TITLE
EPMLABSBRN-176 add SonarCloud integration

### PR DESCRIPTION
**Pull request by Mikhail-Sosnin**
[EPMLABSBRN-176](https://jira.epam.com/jira/browse/EPMLABSBRN-176)

**Description**: SonarCloud integration.
Here we could find the analysis results: https://sonarcloud.io/dashboard?id=Brain-up_brn

Documentation:
- https://docs.sonarqube.org/latest/analysis/languages/java/
- https://docs.sonarqube.org/latest/

### CODE REVIEW

**Comment by ElenaSpb** [comment]
could you please add a section in readme file about sonar work and how every developer can check it report before/after MR

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+sonar.organization=Brain-up
+sonar.projectKey=Brain-up
+
+sonar.sources=src/main/java
```
Comment (goopnigoop) [inline sonar-project.properties]: From my point of view, we have to use here source path to kotlin sources, unfortunately, i have no idea whether sonar has rules for kotlin or not

**Comment by goopnigoop** [comment]
Like that now we have sonar check

**Comment by DmitriyStoyanov** [comment]
Looks like sonarcloud trigger failed because of:

Gradle project detected. You should use the SonarQube plugin for Gradle during build rather than using this GitHub Action.
##[error]Docker run failed with exit code 1

File: sonar-project.properties (hunk 1 above)
Comment (Mikhail-Sosnin) [reply sonar-project.properties]: updated

**Pull request closed without merging**